### PR TITLE
added cvt_cvt-w24-384-in22k_finetuned-in1k_4 model

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds model `cvt_cvt-w24-384-in22k_finetuned-in1k_4` (rank 1 on website) to Brain-Score registry and model bank

### CODE REVIEW

File: brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    layers += ['layernorm']
+    layers = ['cvt.encoder.stages.2.layers.3']
+    return layers
+
+
+def get_bibtex(model_identifier):
+    """
+    A method returning the bibtex reference of the requested model as a string.
+    """
+    return ''
```
Comment (mschrimpf) [inline brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py:42]:
```suggestion
    return """@article{DBLP:journals/corr/abs-2103-15808,
  author       = {Haiping Wu and
                  Bin Xiao and
                  Noel Codella and
                  Mengchen Liu and
                  Xiyang Dai and
                  Lu Yuan and
                  Lei Zhang},
  title        = {CvT: Introducing Convolutions to Vision Transformers},
  journal      = {CoRR},
  volume       = {abs/2103.15808},
  year         = {2021},
  url          = {https://arxiv.org/abs/2103.15808},
  eprinttype    = {arXiv},
  eprint       = {2103.15808},
  timestamp    = {Tue, 18 Oct 2022 08:35:30 +0200},
  biburl       = {https://dblp.org/rec/journals/corr/abs-2103-15808.bib},
  bibsource    = {dblp computer science bibliography, https://dblp.org}
}"""
```

File: brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    check_models.check_base_models(__name__)
+
+"""
+Notes on the error:
+
+- 'channel_x' key error: 
+# 'embeddings.patch_embeddings.projection',
+https://github.com/search?q=repo%3Abrain-score%2Fmodel-tools%20channel_x&type=code
+
+"""
```
Comment (mschrimpf) [inline brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py:135]: ?

File: brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+numpy
+torch
+transformers
+ssl
+PIL
+functools
```
Comment (mschrimpf) [inline brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/requirements.txt:6]:
```suggestion
```
already part of standard python

**Review by mschrimpf — APPROVED** [review]

**Comment by mike-ferguson** [comment]
Closed until validated on Sanghavi Benchmark

**Pull request closed without merging**